From 3199a31536739fe6272ec342ff6443049f55f11c Mon Sep 17 00:00:00 2001
From: Nancy Chen <Nancy.Chen@freescale.com>
Date: Tue, 1 Dec 2009 15:48:30 -0600
Subject: [PATCH] ENGR00118915 MX23: VDDD_BO voltage needs to be updated

MX23: VDDD_BO voltage needs to be updated which caused the system hangs
if cpufreq Conservative governor is using and CPUFREQ is trigged.

Signed-off-by: Nancy Chen <Nancy.Chen@freescale.com>
---
 arch/arm/plat-stmp3xxx/cpufreq.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-stmp3xxx/cpufreq.c b/arch/arm/plat-stmp3xxx/cpufreq.c
index 9dd5a64..69d2f3b 100644
--- a/arch/arm/plat-stmp3xxx/cpufreq.c
+++ b/arch/arm/plat-stmp3xxx/cpufreq.c
@@ -66,12 +66,12 @@ static struct profile {
 	1175000, 173000, 3300000, 1750000, 0 },
 #ifdef CONFIG_STMP378X_RAM_MDDR
 	{  64000,  64000,  48000, 3, 1050000,
-	925000, 150000, 3300000, 1750000, 0 },
+	975000, 150000, 3300000, 1750000, 0 },
 	{  24000,  24000,  24000, 3, 1050000,
 	975000, 150000, 3075000, 1725000, 1 },
 #else
 	{  64000,  64000,  96000, 3, 1050000,
-	925000, 150000, 3300000, 1750000, 0 },
+	975000, 150000, 3300000, 1750000, 0 },
 #endif
 };
 
-- 
1.5.4.4

